app/gui/playwright.config.ts

```diff
@@ -128,19 +128,6 @@ export default defineConfig({
         storageState: path.join(dirName, './playwright/.auth/user.json'),
       },
     },
-    {
-      name: 'Auth',
-      testDir: './integration-test/dashboard/auth',
-      expect: {
-        toHaveScreenshot: { threshold: 0 },
-        timeout: TIMEOUT_MS,
-      },
-      timeout: TIMEOUT_MS,
-      use: {
-        baseURL: `http://localhost:${ports.dashboard}`,
-        actionTimeout: TIMEOUT_MS,
-      },
-    },
```

Comment:
Why is this removed?

Reply:
There are no tests in this project. Agreed with @MrFlashAccount